plugins/services/src/js/containers/service-detail/ServiceDetail.js

```diff
@@ -107,6 +109,15 @@ class ServiceDetail extends mixin(TabsMixin) {
     }
   }
 
+  // will use full screen modals Philipp is making
+  // for now: render into a blank page
```

Comment:
Hey Andrew! I think there's a misunderstanding. 

The ultimate goal was to have full screen modals as just pages. Having the ServiceForm to be a Fullscreen "modal" was a quick and dirty solution. Our initial plan for this refactoring was:

1. Open the Fullscreen modal on a blank page as a first step. Just to cut off the unneeded network activity that happens underneath the modal. 
2. Copy what we have for the app form right now to implement the Package Install Fullscreen form
3. Implement a separate Fullscreen page template with no Sidebar etc. 
4. Refactor these two modals to be pages 

Looks like you guys changed the plan along the way, which is fine. Just wanted to remind you that there's no need in waiting for @Poltergeist if you already have your form as a page. Please continue adding missing stuff like cancel button and review and run.

/cc @MatApple @orlandohohmeier

Reply:
Makes sense. For me, it shouldn't really have an effect of where I render. Either way it'll be fullscreen. I'm working on adding it into the flow right now - using these prototypes: https://mesosphere.invisionapp.com/share/A4DA0J2QH#/screens/251180685 👍